config/dictionaries/resource.json

```diff
@@ -217,6 +218,14 @@
         "endpoint": "/v4/device(<ID>)",
         "filters": "",
         "data": "{\n    \"should_be_running__release\": \"<RELEASE ID>\"\n}"
+      },
+      {
+        "id": "deactivate-device",
+        "summary": "Deactivate an offline device (Includes charges)",
```

Comment:
`Includes charges` isn't clear. Do we have space for `(this will charge your account)`?

Maybe that should even be in capitals...

Reply:
Unfortunately, it spans into two rows.
The best option would be to have a subtitle field in light gray, but implementing it only for this entry would seem odd.

How about 'CHARGEABLE'?

Reply:
Yeah, I think CHARGEABLE is will do, it certainly seems pretty clear to me.